tools/coqmod/dune

```diff
@@ -0,0 +1,18 @@
+(vendored_dirs csexp)
+
+(library
+ (name coqmodlib)
+ (modules :standard \ coqmod)
+ (libraries coq-core.lib csexp))
```

Comment:
This will mean that anybody depending on coqmodlib will also depend on csexp. I think we could factor this further. The csexp stuff for token.ml can be done in another module called serialization.ml or something.